src/classes/UTIL_MasterSchedulableHelper.cls

```diff
@@ -122,22 +122,32 @@ public without sharing class UTIL_MasterSchedulableHelper {
     * @return void
     */
     public static void setScheduledJobs() {
-        //don't auto schedule jobs if the setting tells us not to
-        Error_Settings__c orgErrorSettings = UTIL_CustomSettingsFacade.getOrgErrorSettings();
-        if (orgErrorSettings.Don_t_Auto_Schedule_Default_NPSP_Jobs__c) {
-            return;
-        }
 
-        Set<String> scheduledJobs = new Set<String>();
-        //query for the job names of all scheduled apex
-        for (CronTrigger ct : [SELECT CronJobDetail.Name FROM CronTrigger WHERE CronJobDetail.JobType = '7']) {
-            scheduledJobs.add(ct.CronJobDetail.Name);
+        Map<String, CronTrigger> scheduledJobs = new Map<String, CronTrigger>();
+        
+        //query for all scheduled apex
+        for (CronTrigger ct : [SELECT Id, CronExpression, CronJobDetail.Name, CreatedBy.Name, CreatedBy.IsActive FROM CronTrigger WHERE CronJobDetail.JobType = '7']) {
+            scheduledJobs.put(ct.CronJobDetail.Name, ct);
         }
```

Comment:
doesn't with sharing just change whether you can see a specific record (ie, private vs public vs owned by someone else), rather than changing your permissions (whether you can see a table and field).